### PR TITLE
Add support for DEB packages for Ubuntu 20.04 (focal)

### LINKED ISSUE #8906: Building packages for Ubuntu 20.04 (Focal)
Currently only packages up to Ubuntu 19.10 (Eoan) are available on Packagecloud. Please add the build for Ubuntu 20.04 (Focal).

### PR DESCRIPTION
**Pull request by prologic**
##### Summary

- Fixes #8906

##### Component Name

- area/packaging

##### Test Plan

```#!sh
root@ubuntu2004:~# apt-get update
Get:1 http://10.0.0.109:8000 focal InRelease [2,774 B]
Get:2 http://10.0.0.109:8000 focal/main amd64 Packages [2,120 B]
Hit:3 http://au.archive.ubuntu.com/ubuntu focal InRelease
Hit:4 http://au.archive.ubuntu.com/ubuntu focal-updates InRelease
Hit:5 http://au.archive.ubuntu.com/ubuntu focal-backports InRelease
Hit:6 http://au.archive.ubuntu.com/ubuntu focal-security InRelease
Fetched 4,894 B in 1s (3,315 B/s)
Reading package lists... Done
root@ubuntu2004:~# apt-get install netdata
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following packages will be upgraded:
  netdata
1 upgraded, 0 newly installed, 0 to remove and 63 not upgraded.
1 not fully installed or removed.
Need to get 9,382 kB of archives.
After this operation, 1,024 B disk space will be freed.
Get:1 http://10.0.0.109:8000 focal/main amd64 netdata amd64 1.23.2-49 [9,382 kB]
Fetched 9,382 kB in 0s (97.9 MB/s)
(Reading database ... 108526 files and directories currently installed.)
Preparing to unpack .../netdata_1.23.2-49_amd64.deb ...
Unpacking netdata (1.23.2-49) over (1.23.2-48) ...
Setting up netdata (1.23.2-49) ...
Processing triggers for systemd (245.4-4ubuntu3) ...
root@ubuntu2004:~# ps aux | grep netdata
netdata    58647  1.0  2.6 265432 26924 ?        Ssl  06:48   0:00 /usr/sbin/netdata -D
netdata    58695  0.0  0.6  26756  6448 ?        Sl   06:48   0:00 /usr/sbin/netdata --special-spawn-server
netdata    58889  0.0  0.3   4108  3304 ?        S    06:48   0:00 bash /usr/libexec/netdata/plugins.d/tc-qos-helper.sh 1
root       58893  0.0  0.0   4748   808 ?        S    06:48   0:00 /usr/libexec/netdata/plugins.d/nfacct.plugin 1
netdata    58895  0.5  0.3  53268  3380 ?        S    06:48   0:00 /usr/libexec/netdata/plugins.d/apps.plugin 1
netdata    58898  0.1  1.7 723396 17480 ?        SLl  06:48   0:00 /usr/libexec/netdata/plugins.d/go.d.plugin 1
root       59200  0.0  0.0   6432   736 pts/0    S+   06:48   0:00 grep --color=auto netdata
root@ubuntu2004:~#
```

##### Additional Information

Used this PR as an excuse to create a proper signed repo (_locally_) for
testing purposes using this very nicely written
[repogen](https://github.com/pgaskin/repogen) tool and performing normal
`apt-key add` and `apt-add-repository` operations.

We will use this to migrate off packagecloud and sign our packaged.

Kill two birds with one stone 🤣

### CODE REVIEW

**Comment by MrFreezeex** [comment]
Maybe It would be better to use this PR https://github.com/netdata/netdata/pull/8982 instead ?

**Comment by prologic** [comment]
> Maybe It would be better to use this PR #8982 instead ?

Thanks! Somehow we forgot all about this :/ I'll look at it tomorrow!

**Comment by berendt** [comment]
@prologic Any news?

**Comment by prologic** [comment]
@amoss Okay to merge this? Adds new set of Debian packages for Ubuntu 20.04 (focal). Should have no impact on anything else.

File: contrib/debian/netdata.postinst [hunk 1]
```diff
@@ -59,7 +59,6 @@ case "$1" in
     chmod 4750 /usr/libexec/netdata/plugins.d/slabinfo.plugin
     chmod 4750 /usr/libexec/netdata/plugins.d/cgroup-network
     chmod 4750 /usr/libexec/netdata/plugins.d/nfacct.plugin
-    chmod 4750 /usr/libexec/netdata/plugins.d/ebpf.plugin
```
Comment (MrFreezeex) [inline contrib/debian/netdata.postinst:62]: I guess since #9628 this removal is no longer needed

Comment (prologic) [reply contrib/debian/netdata.postinst:62]: Correct

**Pull request merged**